.gitignore

```diff
@@ -24,8 +24,11 @@ web/.htaccess
 # explicitly.
 web/license.txt
 web/readme.html
-web/index.php
 web/wp-*
+!web/index.php
```

Comment:
`web/index.php` is placed by Composer, so it seems that this one, at a minimum, should remain ignored.

Reply:
Agree 👍 
Updated to restore that line.